hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemConfigurations.java

```diff
@@ -62,6 +62,8 @@
   public static final boolean DEFAULT_OPTIMIZE_FOOTER_READ = false;
   public static final boolean DEFAULT_ALWAYS_READ_BUFFER_SIZE = false;
   public static final int DEFAULT_READ_AHEAD_BLOCK_SIZE = 4 * ONE_MB;
+  // Default value of read ahead range.
+  public static final int DEFAULT_READ_AHEAD_RANGE = 64 * ONE_KB; // 64 KB
```

Comment:
no need for the // comment; it's pretty obvious. 

Why was this range chosen?

Reply:
Ok. This as per suggestion from Rajesh after his experiments. https://github.com/apache/hadoop/pull/2307#issuecomment-694159756